HW1/P3/P3.txt

```diff
@@ -0,0 +1,2 @@
+# Your discussion here
+(u'aeprs', 11, [u'asper', u'pares', u'parse', u'pears', u'prase', u'presa', u'rapes', u'reaps', u'spaer', u'spare', u'spear'])
```

Comment:
There are actually 3 lists with 11 words. I think very minor additions to your code will output them all.